actions/setup-nix/README.md

```diff
@@ -1,32 +1,97 @@
-# setup-nix
+# Setup Nix Action
 
 `setup-nix` installs a nix environment and configures it with the specified
-caches
+caches. Support access to the internal S3 Nix cache in read-only mode and
+read/write mode.
 
 ## Configuration
 
+## Inputs
+
+| Name                    | Description                                                                                        | Required | Default     |
+| ----------------------- | -------------------------------------------------------------------------------------------------- | -------- | ----------- |
+| `cache-url`             | Custom Nix cache URL, for example, `'s3://<bucket-name>'` for an S3-backed cache.                  | Yes      | N/A         |
+| `cache-pubkeys`         | Comma-separated list of custom Nix cache public keys.                                              | Yes      | N/A         |
+| `cache-mode`            | Specify either `'read'` for read-only access or `'push'` for uploading build results to the cache. | Yes      | `read`      |
+| `enable-aws`            | Enable AWS credentials for Nix cache (required for S3-backed caches).                              | No       | `false`     |
```

Comment:
If this is disabled, then the action will just do the install?

Reply:
I was able to separate into 2 actions so now it should be a lot simpler https://github.com/smartcontractkit/.github/pull/502/commits/f903c07a199c1b28a8256c7e10b4c2d2a45f6db7